index.html

```diff
@@ -0,0 +1,738 @@
+
+                            <div class="distract-image">
+                                <div class="overlay3"></div>
+                                    <div class="image"><img src="images/distract.png" alt="mind mapping"></div>
+                                    <div class="group-item-text">
+                                <span class="image-title">MOST POPULAR</span>
+                                <h4 class="article-title"><a href="#">EA rep calls loot boxes 'surprise mechanics' -- no, just no</a></h4>
+                                <span class="article-author-detail"><a href="#">Rachel Kaser</a>7 hours ago</span>
+                            </div>
+                        </div>
+              
+                            <div class="related">
+                                <div class="related-details">
+                                        <h5 class="related-latest">LATEST</h5><br>
+                                        <span class="related-title"><a href="#">A Super Mario battle royale should not be this addictive</a></span>
+            
+              
+                                </div>
+                                <img src="images/latest8.png" class="related-image" alt="">
+                            </div>
+                             <ul class="group-list">
+                                 <li><a href="#">E3 2019: All the release dates you need to know</a></li>
+                                 <li><a href="#">Wikipedia titles you can sing to the 'Teenage Mutant Ninja Turtles' theme song, tweeted</a></li>
+                                 <li><a href="#">New Avengers game hints at a Marvel Gaming Universe</a></li>
+                                 <li><a href="#">Xbox head thinks E3 is about boosting gaming's reputation -- it's not</a></li>
+                                 <li><a href="#">Nintendo shocks at E3 with surprise Zelda sequel announcement</a></li>
+                             </ul>
+                    </div>
+                
+                </div>
+
+
+
+        <!--LATEST DEALS-->
+
+        <div class="main-deals">
+            <h2 class="latest-deals-title"><a href="#">Latest deals <i class="fas fa-greater-than"></i></a></h2>
+            <div class="more-deals"><a href="#">See all Deals</a></div>
+        </div>
+
+        
+
+        <div class="latest-deals">
+
+            <div class="latest-deals-thumbnail">
+                <div class="latest-deals-img">
+                <img src="images/deals-epicpxls.png" alt="Epic deals">
+            </div>
+            <div class="latest-deals-text">
+                <h3 class="deals-title"><a href="#">EpicPxls Design Assets Premium Plan: Lifetime subscription</a></h3>
+                <span class="deals-price"><span class="strike-price">$2400</span>  $99</span>
+            </div>
+            </div>
+
+            <div class="latest-deals-thumbnail">
```

Comment:
Each one of the 8 articles must be inside an `article` tag instead. kindly do that with all of them.